integration-tests/utils/integration-utils.js

```diff
@@ -174,6 +174,28 @@ const withScreenshot = _.curry((testName, fn) => async options => {
   }
 })
 
+const logPageConsoleMessages = page => {
```

Comment:
@dmohs how is one supposed to use this method? I tried using it [here](https://github.com/DataBiosphere/terra-ui/pull/2790#discussion_r793928434), but the console.info calls are not going through line 178.